tests/page-mouse.spec.ts

```diff
@@ -111,7 +111,7 @@ it('should trigger hover state', async ({page, server}) => {
 it('should trigger hover state on disabled button', async ({page, server}) => {
   await page.goto(server.PREFIX + '/input/scrollable.html');
   await page.$eval('#button-6', (button: HTMLButtonElement) => button.disabled = true);
-  await page.hover('#button-6', { timeout: 5000 });
+  await page.hover('#button-6', { timeout: 2000 });
```

Comment:
this is just a drive by to make this test faster right? It doesn't somehow fail with 5000ms now?

Reply:
Yep.